packages/backend/convex/httpActions/clerk.ts

```
@@ -41,8 +43,23 @@ export const clerkHandler = httpAction(async (ctx, request) => {
       await ctx.runMutation(internal.users.mutations.createFromClerk, {
         data: event.data,
       });
-      // throw new Error("supposed to send welcome email here with resend")
-      
+
+      // Send welcome email
+      const primaryEmail =
+        event.data.email_addresses?.find(
+          (email) => email.id === event.data.primary_email_address_id,
+        )?.email_address || event.data.email_addresses?.[0]?.email_address;
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)